clients/src/main/java/org/apache/kafka/clients/consumer/StickyAssignor.java

```diff
@@ -181,187 +169,35 @@
  * Any consumer that uses sticky assignment can leverage this listener like this:
  * <code>consumer.subscribe(topics, new TheNewRebalanceListener());</code>
  *
+ * Note that you can leverage the {@link CooperativeStickyAssignor} to never revoke any partitions to begin with, except
```

Comment:
nit: the phrasing "to never revoke" is a little awkward. Maybe something like this:

> Note that you can leverage {@link CooperativeStickyAssignor} so that only partitions which are being reassigned to another consumer will be revoked.